django_adobesign/backend.py

```diff
@@ -64,20 +64,19 @@ def create_signature(self, signature, post_sign_redirect_url=None,
         self.map_adobe_signer_to_signer(signature)
         return signature
 
-    def update_signer_status(self, signer, status):
-        raise NotImplementedError()
-
     def map_adobe_signer_to_signer(self, signature):
-        # Can raise a Signer.DoesNotExist
+        signers = signature.signers.all()
```

Comment:
```python
signers = {signer.email: signer for signer in signature.signers.all()}
# (...)
signer = signers[adobe_signer['memberInfos'][0]['email'].lower()]
````

Reply:
(except KeyError: ...)

Reply:
@ewjoachim there is also the order param, and he is important because we can have two signer with same email, like an employee and an other signer: we fill in the email as we wish for this other signer